src/Symfony/Component/HttpKernel/Tests/EventListener/SessionListenerTest.php

```diff
@@ -590,6 +590,30 @@ public function testResponseHeadersMaxAgeAndExpiresDefaultValuesIfSessionStarted
         $this->assertFalse($response->headers->has(AbstractSessionListener::NO_AUTO_CACHE_CONTROL_HEADER));
     }
 
+    public function testPrivateResponseMaxAgeIsRespectedIfSessionStarted()
+    {
+        $kernel = $this->createMock(HttpKernelInterface::class);
+
+        $session = $this->createMock(Session::class);
+        $session->expects($this->once())->method('getUsageIndex')->willReturn(1);
+        $request = new Request(server: ['SERVER_PROTOCOL' => 'HTTP/1.0']);
```

Comment:
This should be changed to be compatible with all supported PHP versions (see failing CI)